docs/api.md

```diff
@@ -178,7 +178,7 @@ documented in the [Browser API ⇗](/docs/browser.md) documentation.
 * See [Browser API ⇗](/docs/browser.md)
 
 <a id="destination"></a>
-### `destination` (SonicBoom | WritableStream)
+### `destination` (SonicBoom | WritableStream | String)
 
 Default: `pino.destination(1)` (STDOUT)
```

Comment:
If `FileDescriptor` isn't going to be in the types supported, then this should read: `pino.destination(process.stdout)`.

Reply:
I think it's fine as it is right now, this is equivalent to:

```js
pino(pino.destination(1))
```

What `pino.destination(1)`  returns is an instance of Sonic Boom.